CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 
 ### New
 
+- **General**: Introduce CloudEvents CRD to KEDA ([#3533](https://github.com/kedacore/keda/issues/3533))
```

Comment:
I'd add this as experimental, clarifying in docs that we can introduce breaking changes to be totally transparent and managing user expectations. Creating a CRD is something complex and it could need some time to be stable.
It can give as flexibility for iterating, respecting our deprecation policies
@tomkerkhove  @zroubalik ?

Reply:
I think we did thorough design reviews so think it's fairly stable as we also have API versions and can announce this as a regular feature without experimental status; but I'm open to ideas.

If we make breaking change, we can still bump the API version of the CRD.

Reply:
Actually I would prefer if we add these big changes as experimental, we can find later that something needs to be changed and bumping api version for this is an overkill. We can make it a stable feature right in the subsequent release.